tasks/framework-tools/frameworkSyncToProject.mjs

```diff
@@ -41,7 +41,7 @@ const ignored = [
   /__fixtures__/,
   /__mocks__/,
   /__tests__/,
-  /.test./,
```

Comment:
@jtoar I think we discussed this and agreed this needed a slight update. Is this change in keeping with what you'd expect?

Reply:
Yeah this looks good